third_party/Makefile

```diff
@@ -70,3 +70,14 @@ ifneq (, $(wildcard rccl-tests/Makefile))
 	cd ./rccl-tests && make MPI=1 MPI_HOME=$(MPI_HOME) HIP_HOME=$(HIP_HOME) RCCL_HOME=$(RCCL_HOME) -j
 	cp -v ./rccl-tests/build/* $(SB_MICRO_PATH)/bin/
 endif
+
+# Build rocblas-bench.
+# RocBLAS is released with rocm, like rocm-4.2.0 and so on.
+# The version we use is the released tag which is consistent with the rocm version in the environment or docker.
+# Since it takes several hours to build, avoid to build again if rocblas-bench exsists.
+rocm_rocblas: sb_micro_path
+ifeq (,$(wildcard ./rocBLAS/build/release/clients/staging/rocblas-bench))
```

Comment:
remove ifeq? https://github.com/microsoft/superbenchmark/blob/d512c89f10e946ad0a0df5c50d435859b44b9e84/third_party/Makefile#L36-L37

Reply:
ok, add `if [ -d rocBLAS ]; then rm -rf rocBLAS; fi` and remove ifeq